### ISSUE TITLE
Trying to Convert Stickers from Telegram to Signal results in "LottieAnimation" shutdown

### ISSUE THREAD

**Issue by PauloDiaboli** [laggykiller/sticker-convert#59]
## Bug Description

Trying to convert Stickers from Telegram to Signal results in: `TypeError: LottieAnimation.lottie_shutdown()`

## Steps to reproduce

1. Download sticker-convert-windows.zip (v2.0.15, windows exe)
2. Add necessary credentials
3. Try to download a sticker pack, convert them, and upload them to Signal
4. See the process fail at the conversion step

## Expected behaviour

The sticker pack should be downloaded, converted, and uploaded without any problems.

## Console Output: 

A lot of (as it appears, for each sticker): 
```
TypeError: LottieAnimation.lottie_shutdown() missing 1 required positional argument: 'self'
Exception in thread Thread-94 (compress_thread):
Traceback (most recent call last):
  File "threading.py", line 1038, in _bootstrap_inner
  File "threading.py", line 975, in run
  File "flow.py", line 353, in compress_thread
  File "utils\converter.py", line 116, in convert
  File "utils\converter.py", line 180, in frames_import
  File "utils\converter.py", line 203, in frames_import_lottie
```

**Comment by laggykiller** [comment]
Thanks for your report. Should be fixed in v2.0.16, please check!

**Comment by PauloDiaboli** [comment]
It did, thank you very much. Worked like a charm
